### PR TITLE
Specify a default quantity of one for when one isn't passed back over

### PR DESCRIPTION
**Pull request by dblencowe**
When getting a recurring payment response it appears that PayPal doesn't send a quantity value back.

In order to stop this throwing an error we need to check if qty is set and if not set a quantity of 1. Once Laravel is >= php7 this piece of code should be replaced with `$item['qty'] ?? 1`

### CODE REVIEW

File: src/Traits/PayPalRequest.php [hunk 1]
```diff
@@ -327,10 +327,10 @@ private function createRequestPayload($method)
...
         ], $this->options);
 
-        $this->post = $this->post->merge($config)
-            ->filter(function ($value, $key) use ($method) {
-                return (($method === 'verifyipn') && ($key === 'METHOD')) ?: $value;
-            });
+        $this->post = $this->post->merge($config);
+        if ($method === 'verifyipn') {
+            $this->post->forget('METHOD');
+        }
```
Comment (srmklive) [inline src/Traits/PayPalRequest.php]: Already merged this commit.

Comment (dblencowe) [inline src/Traits/PayPalRequest.php]: Yeah, I'm not sure why it's not picking that up... Any ideas?

File: src/Services/ExpressCheckout.php [hunk 2]
```diff
@@ -73,9 +73,9 @@ protected function setCartItems($items)
     {
         return (new Collection($items))->map(function ($item, $num) {
             return [
-                'L_PAYMENTREQUEST_0_NAME'.$num  => $item['name'],
-                'L_PAYMENTREQUEST_0_AMT'.$num   => $item['price'],
-                'L_PAYMENTREQUEST_0_QTY'.$num   => $item['qty'],
+                'L_PAYMENTREQUEST_0_NAME' . $num => $item['name'],
```
Comment (srmklive) [inline src/Services/ExpressCheckout.php]: CS fix. Need to remove extra space before & after concatenation from line 75-77

File: src/Traits/PayPalRequest.php (hunk 1 above)
Comment (srmklive) [inline src/Traits/PayPalRequest.php]: Probably because i changed the commit message before merge.

**Pull request merged**